src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -217,6 +225,79 @@ private async Task<IEnumerable<SelectListItem>> GetTestUsersForList()
             return userItems;
         }
 
+        private async Task<IEnumerable<SelectListItem>> GetAppsList()
+        {
+            List<SelectListItem> apps = new List<SelectListItem>();
+
+            string path = this._localPlatformSettings.AppRepsitoryBasePath;
+
+            if (!Directory.Exists(path))
+            {
+                return apps;
+            }
+
+            string configPath = path + "config";
+            if (Directory.Exists(configPath))
+            {
+                Application app = GetAppItem(configPath);
+                if (app != null)
+                {
+                    apps.Add(GetSelectItem(app, path));
+                }
+            }
+
+           string[] directories =  Directory.GetDirectories(path);
```

Comment:
Have you considered supporting a folder structure such as 
- repos base folder
  - org 1
    - app A
    - app B
  - org 2
      - app A
      - app B